scaffolds.json

```diff
@@ -11,6 +11,12 @@
     "description": "Scaffold contains required folder structure as well as some example Cadence code.",
     "commit": "f14d43b951400b95e850c07023735538ff836ac6"
   },
+  {
+    "name": "Cadence NFT Project",
+    "repo": "https://github.com/nvdtf/flow-nft-scaffold.git",
+    "description": "Scaffold contains the ExampleNFT sample NFT contract.",
+    "commit": "624ffd71bd2f86faf954d251f69d346ddff1446a"
```

Comment:
does it make sense to be categorized under any of these types: web, mobile, unity?

Reply:
none of web, mobile, unity apply to the scaffold, it's just cadence code importing NonFungibleToken